beacon-chain/execution/log_processing.go

```diff
@@ -32,7 +35,7 @@ var (
 	depositEventSignature = hash.HashKeccak256([]byte("DepositEvent(bytes,bytes,bytes,bytes,bytes)"))
 )
 
-const eth1DataSavingInterval = 1000
+const eth1DataSavingInterval = 10000
```

Comment:
Is this change relevant? Seems like it would change the existing behavior in addition to the new behavior.